package.json

```diff
@@ -9,6 +9,7 @@
     "type": "git",
     "url": "https://github.com/joshwcomeau/guppy.git"
   },
+  "license": "ISC",
```

Comment:
As far as I can tell, there's no convention as to where to place the `license` key in package.json, so I placed it at the bottom of the "header" information (all the informational keys that have no real effect on how the application functions).